Lib/asyncio/graph.py

```diff
@@ -0,0 +1,276 @@
+"""Introspection utils for tasks call graphs."""
+
+import dataclasses
+import sys
+import types
+import typing
```

Comment:
Didn't we remove typing imports recently from `asyncio` to speed-up import time? Because I think importing `asyncio` would now also import `typing` since we do `from .graph import *` in `asyncio.__init__`. I think we should let typeshed the responsibility of having the type hints.

Reply:
I just rebased this PR, so the current source may be outdated with other changes we did elsewhere

Reply:
First off, I don't think the crusade to remove typing imports from the standard library makes sense. Any non-trivial application will import typing anyway through some third-party dependency. It's a lot of effort for a very brittle outcome.

As for this particular import, we need it to type a file. I'll see what I can do to avoid the import.

Reply:
Well.. whether it makes sense or not is a legitimate question, but since something was recently approved to explicitly removed `typing` from asyncio imports, I thought it would have been better not to implicitly revert it (7363476b6405e3d288a61282aa7bc6aca9c2114d).